CoEpi/ui/healthquiz/HealthQuizViewController.swift

```diff
@@ -37,7 +39,8 @@ class HealthQuizViewController: UIViewController, ErrorDisplayer {
 
     override func viewDidLoad() {
         questionList.register(cellClass: QuestionCell.self)
-
+        submitButton.isEnabled = false
```

Comment:
This should be reactive. Add an observable to the view model with enabled status. Derive enabled status from items observable (`selectedSymptoms` is not empty).